aws/resource_aws_iot_role_alias_test.go

```diff
@@ -0,0 +1,332 @@
+package aws
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/iot"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAWSIotRoleAlias_basic(t *testing.T) {
+	alias := acctest.RandomWithPrefix("RoleAlias-")
+	alias2 := acctest.RandomWithPrefix("RoleAlias2-")
+
+	resource.Test(t, resource.TestCase{
```

Comment:
We can use `resource.ParallelTest()` here to enable concurrency with other acceptance testing 👍 

```suggestion
	resource.ParallelTest(t, resource.TestCase{
```